spec/html-proofer/html_spec.rb

```diff
@@ -19,40 +19,40 @@
     expect(proofer.failed_tests).to eq []
   end
 
-  it 'fails for an invalid tag' do
+  it 'works for custom tags' do
     html = "#{FIXTURES_DIR}/html/invalid_tag.html"
     proofer = run_proofer(html, :file, { :check_html => true, :validation => { :report_invalid_tags => true} })
-    expect(proofer.failed_tests.first).to match(/Tag myfancytag invalid \(line 2\)/)
+    expect(proofer.failed_tests).to eq []
   end
 
   it 'fails for an unmatched end tag' do
     html = "#{FIXTURES_DIR}/html/unmatched_end_tag.html"
     proofer = run_proofer(html, :file, { :check_html => true })
-    expect(proofer.failed_tests.first).to match(/Unexpected end tag : div \(line 3\)/)
+    expect(proofer.failed_tests.first).to include("@3:1: That tag isn't allowed here  Currently open tags: html, body..\n</div>\n^ (line 3)")
   end
 
-  it 'fails for an unescaped ampersand in attribute' do
+  it 'allows an unescaped ampersand in attribute' do
     html = "#{FIXTURES_DIR}/html/unescaped_ampersand_in_attribute.html"
     proofer = run_proofer(html, :file, { :check_html => true })
-    expect(proofer.failed_tests.first).to match(/htmlParseEntityRef: expecting ';' \(line 2\)/)
+    expect(proofer.failed_tests).to eq []
   end
 
   it 'fails for mismatch between opening and ending tag' do
     html = "#{FIXTURES_DIR}/html/opening_and_ending_tag_mismatch.html"
     proofer = run_proofer(html, :file, { :check_html => true })
-    expect(proofer.failed_tests.first).to match(/Opening and ending tag mismatch: p and strong/)
+    expect(proofer.failed_tests.first).to include("@3:31: That tag isn't allowed here  Currently open tags: html, body, p, strong..\n<p>The quick <strong>brown fox</p>\n                              ^ (line 3)")
```

Comment:
Hmm. How do you feel about scrubbing the row/column position information in the front for all of these (`@3:31:`). It seems superfluous, given that we report the `line: ` at the end.

Reply:
This is fixed at https://github.com/gjtorikian/html-proofer/pull/362/files#diff-9549a780abfd37b3a235e0289440f6d6R40